jekyll/_cci2/server/installation/phase-2-core-services.adoc

```diff
@@ -0,0 +1,619 @@
+  auth:
+    password: <secret-alphanumeric-password>
+    erlangCookie: <secret-alphanumeric-key>
+----
+
+=== Pusher Secret
+The applications requires a secret for Pusher. CircleCI will not be able to recover the values if lost. Based on how you prefer to manage secrets there are 2 options.
+
+* *Option 1 - Create the Secret yourself*
+
+[source,shell]
+----
+kubectl create secret generic pusher \
+--from-literal=secret=<pusher-secret>
+----
+
+* *Option 2 - CircleCI Creates the secret*
+
+Add the value to values.yaml, and CircleCI will create the secret automatically.
+
+[source,yaml]
+----
+pusher:
+  secret: <pusher-secret>
+----
+
+[#global]
+=== Global
+All values in this section are children of global.
+
+[#circleci-domain-name]
+==== CircleCI domain name (required)
+Enter the domain name you specified when creating your Frontend TLS key and certificate.
+
+[source,yaml]
+----
+global:
+  ...
+  domainName: <domain-name-for-circleci>
+----
+
+[#tls]
+=== TLS
+For TLS, you have 4 options: 
+
+* Do nothing.  https://letsencrypt.org/[Let's Encrypt] will automatically request and manage certificates for you.  This is a good option for trials, it is not recommended for production use.
+
+* You can supply a private key and certificate
++
+You may have created this during the prerequisite steps.  They will need to be base64 encoded. You can retrieve and encode the values with the following commands:
++
+[source,bash]
+----
+cat /etc/letsencrypt/live/<CIRCLECI_SERVER_DOMAIN>/privkey.pem | base64
+cat /etc/letsencrypt/live/<CIRCLECI_SERVER_DOMAIN>/fullchain.pem | base64
+----
++
+Add them to `values.yaml``:
++
+[source,yaml]
+----
+tls:
+  certificate: <full-chain>
+  privateKey: <private-key>
+----
+
+* Have https://docs.aws.amazon.com/acm/latest/userguide/acm-overview.html[AWS Certificate Manager (ACM)] automatically request and manage certificates for you.  Follow the https://docs.aws.amazon.com/acm/latest/userguide/gs-acm-request-public.html[ACM documentation] for instructions on how to generate ACM certificates.
++
+Enable `aws_acm` and add the `service.beta.kubernetes.io/aws-load-balancer-ssl-cert` annotation to point at the ACM ARN
++
+[source,yaml]
+----
+nginx:
+  annotations:
+    service.beta.kubernetes.io/aws-load-balancer-ssl-cert: <acm-arn>
+  aws_acm:
+    enabled: false
+----
++
+[WARNING]
+==== 
+If you have already deployed CircleCI server, enabling ACM is a destructive change to the loadbalancer. The service will have to be regenerated to allow the use of your ACM certificates and so the associated loadbalancer will also be regenerated. 
+You will need to update your DNS records to the new loadbalancer once you have redeployed CircleCI server.
+====
+
+* Disable TLS termination within CircleCI.  The system will still need to be accessed over HTTPS, so TLS termination will be required somewhere upstream of CircleCI.  Implement this by following step 1 (do nothing) and forward to CircleCI on port 80 after terminating TLS.
```

Comment:
@lloydChris is there something they need to add to values.yaml here that we can show?